onnx_chainer/testing/test_onnxruntime.py

```diff
@@ -19,8 +20,16 @@
 
 MINIMUM_OPSET_VERSION = 7
 
+TEST_OUT_DIR = 'out'
+
+
+def check_output(model, x, filename, out_key='prob', opset_version=None):
+    try:
+        os.makedirs(TEST_OUT_DIR)
+    except FileExistsError:
```

Comment:
`makedirs(path, exist_ok=True)` option is same maybe (I'm not completely sure...)

Reply:
Thanks! I forgot we have dropped Python 2 support.